airflow/api/common/mark_tasks.py

```diff
@@ -158,11 +159,13 @@ def set_state(
     qry_dag = get_all_dag_task_query(dag, session, state, task_id_map_index_list, dag_run_ids)
 
     if commit:
+        logger = TaskContextLogger("webserver")
```

Comment:
is this certain to be webserver or might it also be API?

Reply:
It's not certain. It can also come from the REST API.